lighthouse-cli/bin.ts

```diff
@@ -330,8 +330,14 @@ export async function runLighthouse(url: string,
     const chromeLauncher = await getDebuggableChrome(flags)
     const results = await lighthouse(url, flags, config);
 
+    // Filter out artifacts except for screenshots in traces to minimize report size.
+    var traces = results.artifacts.traces;
+    for (var pass in traces)
+      traces[pass]['traceEvents'] = traces[pass]['traceEvents'].filter(e => e['cat'] === 'disabled-by-default-devtools.screenshot');
```

Comment:
I'm in favor of bringing back a screenshots audit but only in thumbnail form and perhaps using SSIM to throttle the exported frames to major changes and not show every loading spinner animation, etc

I can throw together a PR for this if we're interested.